Remove mac-ventura* platform folders, Ventura/+ test expectations.

bls1999 opened the pull request:
#### 41e8735376853d0611d0291cdd7147346d0ff0e1
<pre>
Remove mac-ventura* platform folders, Ventura/+ test expectations.
<a href="https://bugs.webkit.org/show_bug.cgi?id=288208">https://bugs.webkit.org/show_bug.cgi?id=288208</a>
<a href="https://rdar.apple.com/145303446">rdar://145303446</a>

Reviewed by Alexey Proskuryakov.

Because WebKit has deprecated building and testing on macOS Ventura, we no
longer have a need for layout test baselines and expectations that are
Ventura-specific.

This PR removes Ventura-specific layout test expectations and baseline folders
and audits expectations that were marked Ventura+.

* LayoutTests/platform/mac-ventura-wk1/TestExpectations: Removed.
* LayoutTests/platform/mac-ventura-wk1/compositing/repaint/iframes/composited-iframe-with-fixed-background-doc-repaint-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/compositing/repaint/iframes/compositing-iframe-scroll-repaint-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/compositing/repaint/iframes/compositing-iframe-with-fixed-background-doc-repaint-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/inserting/caret-position-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/anchor-element-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attrib-string-colors-with-color-filter-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attribute-string-for-copy-with-color-filter-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-1-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-3-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-4-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-5-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-with-style-1-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-with-style-2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-for-typing-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-for-typing-with-color-filter-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/basic-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/comment-cdata-section-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/font-size-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/font-style-variant-effect-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/font-weight-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/letter-spacing-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/text-decorations-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/vertical-align-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/input/caret-primary-bidi-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/input/firstrectforcharacterrange-plain-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/input/firstrectforcharacterrange-styled-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/selection/5825350-2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/selection/mixed-editability-10-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/fast/forms/datalist/datalist-textinput-appearance-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/fast/replaced/replaced-breaking-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/imported/w3c/web-platform-tests/css/css-tables/border-writing-mode-dynamic-001-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/mime-types/canPlayType-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/TestExpectations: Removed.
* LayoutTests/platform/mac-ventura-wk2/compositing/backing/inline-block-no-backing-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/compositing/backing/whitespace-nodes-no-backing-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/compositing/fixed-image-loading-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/compositing/hidpi-image-backing-store-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/compositing/hidpi-image-backing-store-scaled-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/compositing/tiling/huge-layer-img-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/compositing/visibility/visibility-image-layers-dynamic-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/fast/dynamic/layer-hit-test-crash-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/fast/forms/datalist/datalist-textinput-appearance-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/fast/forms/time/time-input-rendering-basic-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/fast/repaint/4776765-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/fast/repaint/placeholder-after-caps-lock-hidden-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/fast/scrolling/mac/scrollbars/scrollbars-controller-type-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/inspector/canvas/recording-offscreen-webgl-full-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/inspector/canvas/recording-webgl-full-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/scrollingcoordinator/mac/fixed-backgrounds/fixed-background-in-overflow-in-iframe-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/TestExpectations: Removed.
* LayoutTests/platform/mac-ventura/accessibility/datetime/input-datetime-local-label-value-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/accessibility/datetime/input-time-label-value-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/css2.1/t09-c5526c-display-00-e-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/css3/filters/composited-during-animation-layertree-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/caret/caret-position-sideways-lr-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/caret/caret-position-vertical-rl-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/caret/color-span-inside-editable-background-expected.html: Removed.
* LayoutTests/platform/mac-ventura/editing/caret/color-span-inside-editable-expected.html: Removed.
* LayoutTests/platform/mac-ventura/editing/caret/insert-paragraph-does-not-paint-stale-carets-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/input/caret-at-the-edge-of-contenteditable-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/input/caret-at-the-edge-of-input-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/input/editable-container-with-word-wrap-normal-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/inserting/before-after-input-element-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/inserting/multiple-lines-selected-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/pasteboard/4806874-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/pasteboard/input-field-1-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/pasteboard/interchange-newline-2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/3690703-2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/3690703-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/3690719-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/4895428-3-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/4975120-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/block-cursor-overtype-mode-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/caret-in-empty-inline-1-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/caret-in-empty-inline-2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/drag-select-1-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/internal-caret-rect-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/select-from-textfield-outwards-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/simple-line-layout-caret-is-gone-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/block/basic/minheight-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/block/margin-collapse/103-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/clip/outline-overflowClip-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/css/002-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/css/focus-ring-exists-for-search-field-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/css/line-height-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/css/rtl-ordering-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/css/text-overflow-input-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/events/context-no-deselect-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/auto-fill-button/hide-auto-fill-strong-password-viewable-treatment-when-form-is-reset-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/auto-fill-button/input-auto-fill-button-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/auto-fill-button/input-contacts-auto-fill-button-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/auto-fill-button/input-credit-card-auto-fill-button-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/auto-fill-button/input-strong-password-viewable-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/basic-inputs-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/box-shadow-override-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/control-restrict-line-height-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/cursor-at-editable-content-boundary-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/datalist/datalist-searchinput-appearance-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/encoding-test-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/fieldset-align-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/form-element-geometry-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-align-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-bkcolor-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-default-bkcolor-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-focus-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-height-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-preventDefault-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-selection-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-spinbutton-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-spinbutton-up-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-visibility-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-width-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-baseline-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-disabled-color-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-double-click-selection-gap-bug-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-placeholder-visibility-1-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-placeholder-visibility-3-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-spaces-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-table-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-text-click-inside-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-text-click-outside-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-text-double-click-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-text-drag-down-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-text-option-delete-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-text-scroll-left-on-blur-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-text-self-emptying-click-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-text-word-wrap-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-type-text-min-width-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-value-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-width-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/listbox-bidi-align-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/minWidthPercent-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/number/number-appearance-rtl-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/number/number-appearance-spinbutton-disabled-readonly-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/number/number-appearance-spinbutton-layer-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/placeholder-position-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/placeholder-pseudo-style-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/search-cancel-button-style-sharing-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/search-display-none-cancel-button-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/search-input-rtl-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/search-rtl-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/search-styled-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/search-vertical-alignment-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/search/search-padding-cancel-results-buttons-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/search/search-size-with-decorations-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/select-visual-hebrew-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/select-writing-direction-natural-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/tabbing-input-iframe-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/text-control-intrinsic-widths-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/textfield-focus-ring-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/textfield-outline-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/textfield-overflow-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/visual-hebrew-text-field-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/frames/take-focus-from-iframe-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/html/details-no-summary4-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/html/details-open-javascript-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/html/details-open2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/html/details-open4-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/invalid/008-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/lists/dynamic-marker-crash-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/overflow/003-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/overflow/overflow-focus-ring-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/repaint/renderer-destruction-by-invalidateSelection-crash-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/repaint/search-field-cancel-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/repaint/subtree-root-skipped-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/replaced/replaced-breaking-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/replaced/replaced-breaking-mixture-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/table/003-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/table/colspanMinWidth-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/table/spanOverlapRepaint-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/table/text-field-baseline-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/text/basic/generic-family-reset-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/text/canvas-color-fonts/linedash-COLR-expected-mismatch.html: Removed.
* LayoutTests/platform/mac-ventura/fast/text/canvas-color-fonts/stroke-color-COLR-expected.html: Removed.
* LayoutTests/platform/mac-ventura/fast/text/canvas-color-fonts/stroke-color-shadow-COLR-expected.html: Removed.
* LayoutTests/platform/mac-ventura/fast/text/canvas-color-fonts/stroke-color-shadow-ctm-COLR-expected.html: Removed.
* LayoutTests/platform/mac-ventura/fast/text/canvas-color-fonts/stroke-gradient-COLR-expected.html: Removed.
* LayoutTests/platform/mac-ventura/fast/text/drawBidiText-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/text/international/bidi-listbox-atsui-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/text/international/bidi-listbox-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/text/international/bidi-menulist-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/text/international/pop-up-button-text-alignment-and-direction-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/text/international/unicode-bidi-plaintext-in-textarea-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/text/resources/Ahem-COLR.ttf: Removed.
* LayoutTests/platform/mac-ventura/fast/text/textIteratorNilRenderer-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/transforms/transformed-focused-text-input-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/url/url-hostname-encoding-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/http/tests/navigation/javascriptlink-frames-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/http/wpt/mediarecorder/mimeType-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-text-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/mime-types/canPlayType-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/imported/w3c/web-platform-tests/url/toascii.window-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/imported/w3c/web-platform-tests/xhr/open-method-case-sensitive-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/media/media-can-play-h265-video-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/media/media-can-play-mpeg4-video-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/media/media-can-play-wav-audio-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/media/video-source-type-params-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/printing/print-with-media-query-destory-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/scrollbars/overflow-scrollbar-combinations-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/svg/custom/inline-svg-in-xhtml-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/svg/custom/svg-fonts-in-html-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/svg/hixie/mixed/003-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug1188-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug12384-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug18359-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug24200-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug2479-2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug2479-3-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug2479-4-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug28928-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug4382-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug4527-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug46368-1-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug46368-2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug51037-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug55545-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug59354-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug7342-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug96334-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug99948-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/dom/tableDom-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/other/move_row-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla_expected_failures/bugs/bug2479-5-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla_expected_failures/bugs/bug92647-1-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/transforms/3d/general/perspective-non-layer-expected.txt: Removed.
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/platform/mac-wk2/TestExpectations:
* LayoutTests/platform/mac/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/290956@main">https://commits.webkit.org/290956@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/2ffd5494fa12b3af85d5e875c820ad8f3f222621

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/91582 "Passed style check") | [  ~~🛠 ios~~](https://ews-build.webkit.org/#/builders/131/builds/11113 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/647 "Built successfully") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/5/builds/96548 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 win~~](https://ews-build.webkit.org/#/builders/59/builds/42263 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [  ~~🛠 ios-sim~~](https://ews-build.webkit.org/#/builders/130/builds/11490 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/123/builds/19542 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/5/builds/96548 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/59/builds/42263 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/94583 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/130/builds/11490 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/82979 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/5/builds/96548 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/130/builds/11490 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/135/builds/568 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wpe-cairo~~](https://ews-build.webkit.org/#/builders/65/builds/41434 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/130/builds/11490 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/136/builds/571 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/2/builds/98555 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 vision~~](https://ews-build.webkit.org/#/builders/128/builds/18740 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/123/builds/19542 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/2/builds/98555 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 vision-sim~~](https://ews-build.webkit.org/#/builders/121/builds/18994 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/8/builds/78817 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/2/builds/98555 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [⏳ 🧪 vision-wk2 ](https://ews-build.webkit.org/#/builders/visionOS-2-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [  ~~🧪 mac-intel-wk2~~](https://ews-build.webkit.org/#/builders/137/builds/437 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/11865 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/14497 "Built successfully and passed tests") | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/127/builds/18734 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/125/builds/18444 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/129/builds/21904 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/124/builds/20210 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

aproskuryakov commented on LayoutTests/TestExpectations:
```diff
@@ -1472,21 +1472,6 @@ fast/images/page-wide-animation-toggle.html [ Skip ]
-http/wpt/service-workers/push-console-messages-no-showNotification.https.html [ Skip ]
-http/wpt/service-workers/push-console-messages-showNotification-async.https.html [ Skip ]
-http/wpt/service-workers/push-console-messages-showNotification-late.https.html [ Skip ]
-http/wpt/service-workers/push-console-messages-showNotification-sync.https.html [ Skip ]
```
This doesn't look quite right. We re-enable these tests in `LayoutTests/platform/mac-wk2` and `LayoutTests/platform/glib`, but removing this block also enables them in mac-wk1, for example. And if you are removing this block, then the Pass expectations in mac-wk2 are superfluous.

Please take a closer look at these tests.

bls1999 replied:
Good point -- I assumed the comment was correct, but I see the nuance. I'll just put these back until we have a chance to do some more scrutiny.

aproskuryakov commented on LayoutTests/platform/mac-wk1/TestExpectations:
```diff
@@ -2594,28 +2594,28 @@ imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/b
+media/track/track-delete-during-setup.html  [ Crash ]
+media/track/track-in-band-cues-added-once.html  [ Crash ]
+media/track/track-in-band.html  [ Crash ]
+media/track/track-long-captions-file.html  [ Crash ]
```
While this change is mechanically correct, now is a good opportunity to sanity check such "always crashes" expectations. I checked `media/track/track-in-band.html`, and it is not crashing.

bls1999 replied:
I was actually in the process of doing this when you commented :) I believe my subsequent commit actioned this feedback.

aproskuryakov commented on LayoutTests/platform/mac-wk1/TestExpectations:
```diff
@@ -2594,28 +2594,28 @@ imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/b
+media/track/track-long-captions-file.html  [ Crash ]
 imported/w3c/web-platform-tests/secure-contexts/basic-popup-and-iframe-tests.html [ Pass Failure ]
-[ Ventura+ ] http/wpt/webaudio/audioworklet-addModule-cors.sub.https.html
+http/wpt/webaudio/audioworklet-addModule-cors.sub.https.html
```
It's sad that we've never went back to file bugs for so many Ventura regressions.

aproskuryakov commented on LayoutTests/TestExpectations:
```diff
@@ -2479,9 +2479,6 @@ fast/images/webp-as-image.html [ Skip ]
 fast/images/animated-webp.html [ Skip ]
 
-# AVIF overlay images are only supported on macOS and iOS post Ventura.
-fast/images/image-overlay-mutiple-frames.html [ Skip ]
```
Assuming this comment is correct, this is going to fail on non-Apple ports.

bls1999 replied:
Good catch! Fixed in latest commit, thanks.

aproskuryakov commented on LayoutTests/platform/mac-wk1/TestExpectations:
```diff
@@ -2594,28 +2594,19 @@ imported/w3c/web-platform-tests/websockets/interfaces/WebSocket/bufferedAmount/b
-[ Ventura+ ] http/wpt/webaudio/audioworklet-addModule-cors.sub.https.html
+[ arm64 ] svg/transforms/transformed-text-fill-pattern.html [ ImageOnlyFailure ]
+media/modern-media-controls/media-controller/media-controller-auto-hide-rewind-with-mouse-enter.html [ Pass Crash ]
+media/track/track-in-band-cues-added-once.html [ Pass Crash ]
```
Not a blocker for landing this, but a some point soon we should make certain that there are bugs filed at least for crashes, if not for all of the regressions.

aproskuryakov commented on LayoutTests/platform/mac-wk1/TestExpectations:
```diff
@@ -2897,9 +2878,9 @@ imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/properties
 [ Sonoma+ ] imported/w3c/web-platform-tests/worklets/paint-worklet-credentials.https.html [ Pass Failure ]
 
 # webkit.org/b/272115 REGRESSION (274826@main): [ Sonoma WK1 ] 2X imported/w3c/web-platform-tests/mixed-content/tentative/autoupgrades tests are flaky failures
```
Please update the comment with new title.

aproskuryakov commented on LayoutTests/platform/mac-wk1/TestExpectations:
```diff
@@ -2910,47 +2891,35 @@ imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/properties
-[ Ventura ] imported/w3c/web-platform-tests/worklets/audio-worklet-referrer.https.html [ Pass Failure ] 
-[ Ventura ] imported/w3c/web-platform-tests/worklets/layout-worklet-credentials.https.html [ Pass Failure ]
-[ Ventura ] imported/w3c/web-platform-tests/worklets/paint-worklet-credentials.https.html [ Pass Failure ]
-[ Ventura+ ] imported/w3c/web-platform-tests/workers/modules/dedicated-worker-import-referrer.html [ Pass Failure ]
+imported/w3c/web-platform-tests/workers/modules/dedicated-worker-import-referrer.html [ Pass Failure ]
```
This needs a new bug, as the remaining test is unrelated to worklet tests.

aproskuryakov commented on LayoutTests/platform/mac-wk1/TestExpectations:
```diff
@@ -2910,47 +2891,35 @@ imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/properties
-# rdar://65188503
- platform/mac/media/media-source/is-type-supported-vp9-codec-check.html [ Skip ]
-
 # rdar://65188503
+platform/mac/media/media-source/is-type-supported-vp9-codec-check.html [ Skip ]
 platform/mac/media/mediacapabilities/vp9-decodingInfo-sw.html [ Skip ]
```
This is supposed to have been fixed long ago.

aproskuryakov commented on LayoutTests/platform/mac-wk1/TestExpectations:
```diff
@@ -2910,47 +2891,35 @@ imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/properties
 
 webkit.org/b/276145 http/tests/security/block-popup-to-all-zero-address.html [ Skip ]
 
+# webkit.org/b/255427 [ Ventura ] 4/13 batch mark expectations and branch bugs
```
This bug is in FIXED state.

aproskuryakov reviewed (approved):
I spot checked some of the issues, and clearly these expectations would benefit from a couple more cleanup passes. Such cleanup shouldn't be blocking removing Ventura expectations, but please do address at least the few specific ones that I called out.

bls1999 committed:
Remove mac-ventura* platform folders, Ventura/+ test expectations.
https://bugs.webkit.org/show_bug.cgi?id=288208
rdar://145303446

Reviewed by Alexey Proskuryakov.

Because WebKit has deprecated building and testing on macOS Ventura, we no
longer have a need for layout test baselines and expectations that are
Ventura-specific.

This PR removes Ventura-specific layout test expectations and baseline folders
and audits expectations that were marked Ventura+.

* LayoutTests/platform/mac-ventura-wk1/TestExpectations: Removed.
* LayoutTests/platform/mac-ventura-wk1/compositing/repaint/iframes/composited-iframe-with-fixed-background-doc-repaint-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/compositing/repaint/iframes/compositing-iframe-scroll-repaint-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/compositing/repaint/iframes/compositing-iframe-with-fixed-background-doc-repaint-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/inserting/caret-position-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/anchor-element-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attrib-string-colors-with-color-filter-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attribute-string-for-copy-with-color-filter-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-1-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-3-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-4-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-5-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-with-style-1-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-across-shadow-boundaries-with-style-2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-for-typing-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/attributed-string-for-typing-with-color-filter-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/basic-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/comment-cdata-section-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/font-size-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/font-style-variant-effect-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/font-weight-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/letter-spacing-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/text-decorations-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/attributed-string/vertical-align-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/input/caret-primary-bidi-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/input/firstrectforcharacterrange-plain-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/mac/input/firstrectforcharacterrange-styled-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/selection/5825350-2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/editing/selection/mixed-editability-10-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/fast/forms/datalist/datalist-textinput-appearance-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/fast/replaced/replaced-breaking-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/imported/w3c/web-platform-tests/css/css-tables/border-writing-mode-dynamic-001-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk1/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/mime-types/canPlayType-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/TestExpectations: Removed.
* LayoutTests/platform/mac-ventura-wk2/compositing/backing/inline-block-no-backing-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/compositing/backing/whitespace-nodes-no-backing-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/compositing/fixed-image-loading-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/compositing/hidpi-image-backing-store-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/compositing/hidpi-image-backing-store-scaled-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/compositing/tiling/huge-layer-img-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/compositing/visibility/visibility-image-layers-dynamic-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/fast/dynamic/layer-hit-test-crash-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/fast/forms/datalist/datalist-textinput-appearance-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/fast/forms/time/time-input-rendering-basic-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/fast/repaint/4776765-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/fast/repaint/placeholder-after-caps-lock-hidden-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/fast/scrolling/mac/scrollbars/scrollbars-controller-type-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/inspector/canvas/recording-offscreen-webgl-full-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/inspector/canvas/recording-webgl-full-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2/scrollingcoordinator/mac/fixed-backgrounds/fixed-background-in-overflow-in-iframe-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/TestExpectations: Removed.
* LayoutTests/platform/mac-ventura/accessibility/datetime/input-datetime-local-label-value-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/accessibility/datetime/input-time-label-value-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/css2.1/t09-c5526c-display-00-e-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/css3/filters/composited-during-animation-layertree-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/caret/caret-position-sideways-lr-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/caret/caret-position-vertical-rl-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/caret/color-span-inside-editable-background-expected.html: Removed.
* LayoutTests/platform/mac-ventura/editing/caret/color-span-inside-editable-expected.html: Removed.
* LayoutTests/platform/mac-ventura/editing/caret/insert-paragraph-does-not-paint-stale-carets-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/input/caret-at-the-edge-of-contenteditable-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/input/caret-at-the-edge-of-input-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/input/editable-container-with-word-wrap-normal-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/inserting/before-after-input-element-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/inserting/multiple-lines-selected-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/pasteboard/4806874-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/pasteboard/input-field-1-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/pasteboard/interchange-newline-2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/3690703-2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/3690703-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/3690719-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/4895428-3-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/4975120-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/block-cursor-overtype-mode-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/caret-in-empty-inline-1-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/caret-in-empty-inline-2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/drag-select-1-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/internal-caret-rect-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/select-from-textfield-outwards-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/editing/selection/simple-line-layout-caret-is-gone-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/block/basic/minheight-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/block/margin-collapse/103-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/clip/outline-overflowClip-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/css/002-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/css/focus-ring-exists-for-search-field-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/css/line-height-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/css/rtl-ordering-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/css/text-overflow-input-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/events/context-no-deselect-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/auto-fill-button/hide-auto-fill-strong-password-viewable-treatment-when-form-is-reset-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/auto-fill-button/input-auto-fill-button-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/auto-fill-button/input-contacts-auto-fill-button-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/auto-fill-button/input-credit-card-auto-fill-button-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/auto-fill-button/input-strong-password-viewable-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/basic-inputs-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/box-shadow-override-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/control-restrict-line-height-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/cursor-at-editable-content-boundary-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/datalist/datalist-searchinput-appearance-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/encoding-test-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/fieldset-align-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/form-element-geometry-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-align-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-bkcolor-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-default-bkcolor-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-focus-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-height-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-preventDefault-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-selection-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-spinbutton-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-spinbutton-up-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-visibility-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-appearance-width-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-baseline-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-disabled-color-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-double-click-selection-gap-bug-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-placeholder-visibility-1-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-placeholder-visibility-3-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-spaces-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-table-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-text-click-inside-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-text-click-outside-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-text-double-click-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-text-drag-down-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-text-option-delete-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-text-scroll-left-on-blur-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-text-self-emptying-click-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-text-word-wrap-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-type-text-min-width-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-value-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/input-width-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/listbox-bidi-align-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/minWidthPercent-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/number/number-appearance-rtl-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/number/number-appearance-spinbutton-disabled-readonly-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/number/number-appearance-spinbutton-layer-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/placeholder-position-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/placeholder-pseudo-style-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/search-cancel-button-style-sharing-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/search-display-none-cancel-button-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/search-input-rtl-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/search-rtl-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/search-styled-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/search-vertical-alignment-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/search/search-padding-cancel-results-buttons-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/search/search-size-with-decorations-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/select-visual-hebrew-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/select-writing-direction-natural-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/tabbing-input-iframe-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/text-control-intrinsic-widths-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/textfield-focus-ring-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/textfield-outline-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/textfield-overflow-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/forms/visual-hebrew-text-field-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/frames/take-focus-from-iframe-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/html/details-no-summary4-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/html/details-open-javascript-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/html/details-open2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/html/details-open4-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/invalid/008-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/lists/dynamic-marker-crash-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/overflow/003-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/overflow/overflow-focus-ring-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/repaint/renderer-destruction-by-invalidateSelection-crash-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/repaint/search-field-cancel-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/repaint/subtree-root-skipped-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/replaced/replaced-breaking-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/replaced/replaced-breaking-mixture-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/table/003-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/table/colspanMinWidth-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/table/spanOverlapRepaint-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/table/text-field-baseline-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/text/basic/generic-family-reset-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/text/canvas-color-fonts/linedash-COLR-expected-mismatch.html: Removed.
* LayoutTests/platform/mac-ventura/fast/text/canvas-color-fonts/stroke-color-COLR-expected.html: Removed.
* LayoutTests/platform/mac-ventura/fast/text/canvas-color-fonts/stroke-color-shadow-COLR-expected.html: Removed.
* LayoutTests/platform/mac-ventura/fast/text/canvas-color-fonts/stroke-color-shadow-ctm-COLR-expected.html: Removed.
* LayoutTests/platform/mac-ventura/fast/text/canvas-color-fonts/stroke-gradient-COLR-expected.html: Removed.
* LayoutTests/platform/mac-ventura/fast/text/drawBidiText-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/text/international/bidi-listbox-atsui-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/text/international/bidi-listbox-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/text/international/bidi-menulist-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/text/international/pop-up-button-text-alignment-and-direction-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/text/international/unicode-bidi-plaintext-in-textarea-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/text/resources/Ahem-COLR.ttf: Removed.
* LayoutTests/platform/mac-ventura/fast/text/textIteratorNilRenderer-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/transforms/transformed-focused-text-input-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/fast/url/url-hostname-encoding-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/http/tests/navigation/javascriptlink-frames-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/http/wpt/mediarecorder/mimeType-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-text-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/imported/w3c/web-platform-tests/html/semantics/embedded-content/media-elements/mime-types/canPlayType-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/imported/w3c/web-platform-tests/url/toascii.window-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/imported/w3c/web-platform-tests/xhr/open-method-case-sensitive-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/media/media-can-play-h265-video-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/media/media-can-play-mpeg4-video-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/media/media-can-play-wav-audio-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/media/video-source-type-params-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/printing/print-with-media-query-destory-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/scrollbars/overflow-scrollbar-combinations-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/svg/custom/inline-svg-in-xhtml-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/svg/custom/svg-fonts-in-html-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/svg/hixie/mixed/003-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug1188-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug12384-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug18359-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug24200-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug2479-2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug2479-3-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug2479-4-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug28928-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug4382-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug4527-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug46368-1-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug46368-2-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug51037-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug55545-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug59354-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug7342-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug96334-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/bugs/bug99948-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/dom/tableDom-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla/other/move_row-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla_expected_failures/bugs/bug2479-5-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/tables/mozilla_expected_failures/bugs/bug92647-1-expected.txt: Removed.
* LayoutTests/platform/mac-ventura/transforms/3d/general/perspective-non-layer-expected.txt: Removed.
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/platform/mac-wk2/TestExpectations:
* LayoutTests/platform/mac/TestExpectations:

Canonical link: https://commits.webkit.org/290956@main
